MAINTAINING.md

```diff
@@ -0,0 +1,9 @@
+<h1 style="border-bottom: 0;">Cairo-lint Maintenance</h1>
```

Comment:
Same as above, we don't precisely know how will the procedure will look like. It will change after refactors and setting up proper CI